src/LazyLoad.jsx

```diff
@@ -157,5 +154,3 @@ LazyLoad.defaultProps = {
   offsetVertical: 0,
   throttle: 250,
 };
-
-module.exports = LazyLoad;
```

Comment:
We still have to use commonjs syntax here, for those who don't use babel. For prettier import flow
`require('react-lazy-load')` instead of `require('react-lazy-load').default`